tempesta_fw/http_parser.c

```diff
@@ -644,7 +641,7 @@ good_looking_eol:							\
 	 * The next chunk of chunked-body payload (RFC 7320, 4.1.2)	\
 	 * needs to be handled.						\
 	 */								\
-	__FSM_JMP(RGen_Body);						\
+	__FSM_JMP(RGen_BodyChunk);					\
```

Comment:
Ok, my previous comment for the place seems invalid. However, it'd be good to rename `RGen_BodyChunk` since now it looks like processing chunked body, but in fact the state also handles chunked messages as well ans `Content-Length` defined. Or you can just write a comment just before the state clarifying the state logic.